src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,27 @@
+import { UserInfo } from '../UserInfo/UserInfo';
+
+import { Props } from '../../types/Props';
+
+import './TodoInfo.scss';
+
+export const TodoInfo: React.FC<Props> = ({ title, completed, user }) => (
+  <>
+    <h2 className="todo__title">{title}</h2>
+    {completed
+      ? (
+        <p className="todo__complited todo__complited--true">
+          Complited
```

Comment:
Fix this typo everywhere:
```suggestion
          Completed
```